### PR TITLE
Adds manage embargo bulk action.

### LINKED ISSUE #2671: Add "Manage embargo" bulk action
This bulk action goes with #2009 (the single-object case).

The goal is to allow users to:

- Create embargoes for a set of objects
- Update embargoes for a set of objects

**Example use case**

University Archives has a set of meeting minutes that need to be embargoed for 15 years from date of the meeting. Each meeting has its own druid. They would like to set the embargo by supplying a list of

- druids
- release dates
- rights to be applied upon release

Example: https://jirasul.stanford.edu/jira/browse/LEGACY-4765

**Functionality**

This would be best done via CSV upload rather than through a form, since the bulk use case is one where the data is likely to vary (different dates or rights for different druids).

Headers:

- druid
- release_date (in YYYY-MM-DD format)
- rights

All three fields are required. Also, adding or extending an embargo requires versioning, so objects either need to be open already and/or opened automatically by running this action.

**Removing an embargo**

In the beginning, I don't think we need to support lifting an embargo via the bulk action by simply removing it. Instead, we should retain the existing practice where an embargo is lifted via setting it to release on the current day. But I could be convinced otherwise.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #2671

## Why was this change made?
Support embargo management from Argo.

## How was this change tested?
Unit, QA

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

**Comment by justinlittman** [comment]
Awaiting further clarification of requirements for embargo rights.

**Comment by justinlittman** [comment]
Tested on QA by @andrewjbtw

File: app/jobs/manage_embargoes_job.rb [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      open_new_version(cocina_object.externalIdentifier, cocina_object.version, msg) unless state_service.allows_modification?
+
+      changes = {}
+      changes[:embargo_release_date] = embargo_release_date if embargo_release_date
+      changes[:embargo_access] = rights if rights
+      unless changes.empty?
+        change_set = ItemChangeSet.new(cocina_object)
+        change_set.validate(changes)
+        change_set.save_model
+        # ItemChangeSet.new(cocina_object).validate(changes).save_model unless changes.empty?
```
Comment (jcoyne) [inline app/jobs/manage_embargoes_job.rb:88]: cruft?

File: app/jobs/manage_embargoes_job.rb [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      state_service = StateService.new(current_druid, version: cocina_object.version)
+      open_new_version(cocina_object.externalIdentifier, cocina_object.version, msg) unless state_service.allows_modification?
+
+      changes = {}
+      changes[:embargo_release_date] = embargo_release_date if embargo_release_date
+      changes[:embargo_access] = rights if rights
+      unless changes.empty?
+        change_set = ItemChangeSet.new(cocina_object)
+        change_set.validate(changes)
+        change_set.save_model
```
Comment (jcoyne) [inline app/jobs/manage_embargoes_job.rb:87]: You should call `save` here, `save_model` is an internal method called by save.

**Review by jcoyne — CHANGES_REQUESTED** [review]

**Review by jcoyne — APPROVED** [review]

**Pull request merged**